### PR TITLE
:ambulance: Message for Orders with Invoiced Status. closes 1122

### PR DESCRIPTION
**Pull request by KolushovAlexandr**
supersede #1143

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Does not work

**Comment by ImplantiSolucoes** [comment]
> supersede #1143

Thanks for helping me with my pull request

> Does not work

It's working, but only works using the order reference:
Screenshot_3
Screenshot_4
Screenshot_5
Screenshot_6
Screenshot_7

**Comment by KolushovAlexandr** [comment]
@ImplantiSolucoes Thanks for contributing! I made some changes so your pr satisfies our company rules

**Comment by ImplantiSolucoes** [comment]
It was very good, I wanted to understand a little javascript like you

File: pos_orders_history/doc/changelog.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.. _changelog:
+`1.3.0`
+-------
 
-Updates
-=======
+**Improvement:** Show Invoiced Orders option
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history/doc/changelog.rst:4]: :+1: checked

File: pos_orders_history_reprint/doc/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,12 @@
 Updates
 =======
 
+`1.0.5`
+--------
+
+- **Fix:** Correcting "There is No Ticket for the Order" Message for Orders with Invoiced Status
+
+
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_orders_history_reprint/doc/changelog.rst:11]: :+1: checked

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by ImplantiSolucoes — APPROVED** [review]
LGTM, tested

**Review by em230418 — CHANGES_REQUESTED** [review]
Тесты не запускаются автоматически. Такая-же история, как и в ветке постарше
https://github.com/it-projects-llc/pos-addons/pull/1100#pullrequestreview-293051886

**Comment by em230418** [comment]
Договорились на том, что тесты будем в рамках другой задачи

**Review by em230418 — APPROVED** [review]

**Pull request merged**